crates/matrix-sdk/src/sync.rs

```diff
@@ -103,37 +103,59 @@ pub enum RoomUpdate {
         room: Room,
         /// Updates to the room.
         updates: LeftRoom,
+        /// Collection of ambiguity changes that room member events trigger.
+        ///
+        /// This is a map of event ID of the `m.room.member` event to the
+        /// details of the ambiguity change.
+        ambiguity_changes: BTreeMap<OwnedEventId, AmbiguityChange>,
```

Comment:
I would personally put `ambiguity_changes` inside `LeftRoom`. Thoughts?

Reply:
Oh right, I agree. I thought `*Room`s were the types from Ruma. Although, in `SyncResponse`, that would make the ambiguity changes available in two places: in the root `ambiguity_changes` field, and for each room. But we can always remove the root field.

Reply:
Done in the latest commit